procedures: Viewing Che workspaces logs (#2504)
codenvy/che-docs: modules/end-user-guide/pages/viewing-che-workspaces-logs.adoc

:_content-type: REFERENCE
:description: Viewing {prod-short} workspaces logs
:keywords: administration-guide, viewing-che-workspaces-logs
:navtitle: Viewing {prod-short} workspaces logs
:page-aliases: .:viewing-che-workspaces-logs.adoc, administration-guide:viewing-che-workspaces-logs.adoc

[id="viewing-{prod-id-short}-workspaces-logs"]
= Viewing {prod-short} workspaces logs

> You can view {prod-short} workspaces logs to get a better understanding and insight into the processes, such as:

Reviewer: ```suggestion
You can view {prod-short} logs to better understand and debug background processes should a problem occur.
```

> You can observe {prod-short} workspaces logs to get a better understanding and insight over the processes, such as:

Reviewer 2: ```suggestion
You can view {prod-short} workspaces logs to get a better understanding and insight into the processes, such as:
```

> Editor logs::

Reviewer: ```suggestion
An issue with an extension::
```

> An issue with an extension::

Reviewer: How about...

```suggestion
An IDE extension misbehaves or needs debugging::
```

> The editor logs list plugins loaded by the editor.

Reviewer: ```suggestion
The logs list the plugins that have been loaded by the editor.
```

> The editor logs lists plugins loaded by the editor.

Reviewer 2: ```suggestion
The editor logs list plugins loaded by the editor.
```

The container runs out of memory::

> The logs contain `OOMKilled`.

Reviewer 2: ```suggestion
The logs contain an `OOMKilled`error message.
```

> The logs contain an `OOMKilled`error message.

Reviewer: ```suggestion
The logs contain an `OOMKilled` error message.
```

> The processes in the container requested more memory than is available in the container.

Reviewer: ```suggestion
Processes running in the container attempted to request more memory than is available.
```

> A process inside the container runs out of memory without the container noticing this::
> The logs contain an error message such as `OutOfMemoryException`.
> A process ran out of memory.

Reviewer 2: ```suggestion
A process runs out of memory::
The logs contain an error message such as `OutOfMemoryException`.
A process inside the container ran out of memory without the container noticing.
```
